modules/audit-enqueued-assets/load.php

```diff
@@ -0,0 +1,210 @@
+		$list_enqueued_styles = get_transient( 'aea_enqueued_styles' );
+		$enqueued_styles      = count( $list_enqueued_styles );
+	}
+	return $enqueued_styles;
+}
+
+/**
+ * Adds tests to site health.
+ *
+ * @since 1.0.0
+ *
+ * @param array $tests Site Health Tests.
+ * @return array
+ */
+function aea_add_enqueued_assets_test( $tests ) {
+	$tests['direct']['enqueued_js_assets']  = array(
+		'label' => esc_html__( 'JS assets', 'performance-lab' ),
+		'test'  => 'aea_enqueued_js_assets_test',
+	);
+	$tests['direct']['enqueued_css_assets'] = array(
+		'label' => esc_html__( 'CSS assets', 'performance-lab' ),
+		'test'  => 'aea_enqueued_css_assets_test',
+	);
+
+	return $tests;
+}
+add_filter( 'site_status_tests', 'aea_add_enqueued_assets_test' );
+
+/**
+ * Callback for enqueued_js_assets test.
+ *
+ * @since 1.0.0
+ *
+ * @return array
+ */
+function aea_enqueued_js_assets_test() {
+	/**
+	 * If the test didn't run yet, deactivate.
+	 */
+	$enqueued_scripts = aea_get_total_enqueued_scripts();
+	if ( false === $enqueued_scripts ) {
+		return array();
+	}
+
+	$result = array(
+		'label'       => esc_html__( 'Enqueued JS assets', 'performance-lab' ),
+		'status'      => 'good',
+		'badge'       => array(
+			'label' => esc_html__( 'Performance', 'performance-lab' ),
+			'color' => 'blue',
+		),
+		'description' => sprintf(
+			'<p>%s</p>',
+			esc_html__( 'The amount of enqueued JS assets is acceptable.', 'performance-lab' )
+		),
+		'actions'     => '',
+		'test'        => 'enqueued_js_assets',
+	);
+
+	if ( $enqueued_scripts > 10 ) {
+		$result['status']         = 'recommended';
+		$result['badge']['color'] = 'orange';
+		$result['description']    = sprintf(
+			/* translators: %s: Number of enqueued scripts */
+			esc_html__( 'Your website enqueues %s scripts. Try to reduce the number of JS assets, or to concatenate them.', 'performance-lab' ),
+			$enqueued_scripts
+		);
+		$result['actions'] .= sprintf(
+			/* translators: 1: HelpHub URL. 2: Link description. */
+			'<p><a target="_blank" href="%1$s">%2$s</a></p>',
+			esc_url( __( 'https://wordpress.org/support/article/optimization/', 'performance-lab' ) ),
+			esc_html__( 'More info about performance optimization', 'performance-lab' )
+		);
+	}
+
+	return $result;
+}
+
+/**
+ * Callback for enqueued_css_assets test.
+ *
+ * @since 1.0.0
+ *
+ * @return array
+ */
+function aea_enqueued_css_assets_test() {
+	/**
+	 * If the test didn't run yet, deactivate.
+	 */
+	$enqueued_styles = aea_get_total_enqueued_styles();
+	if ( false === $enqueued_styles ) {
+		return array();
+	}
+
+	$result = array(
+		'label'       => esc_html__( 'Enqueued CSS assets', 'performance-lab' ),
+		'status'      => 'good',
+		'badge'       => array(
+			'label' => esc_html__( 'Performance', 'performance-lab' ),
+			'color' => 'blue',
+		),
+		'description' => sprintf(
+			'<p>%s</p>',
+			esc_html__( 'The amount of enqueued CSS assets is acceptable.', 'performance-lab' )
+		),
+		'actions'     => '',
```

Comment:
in all the actions blocks
Can we add a button/link to reset the count
I can see that someone could be trying to improve the score but it not changing as the transient set to 1 day :-)